web/src/timerender.ts

```diff
@@ -215,13 +215,8 @@ export function render_now(time: Date, today = new Date()): TimeRender {
 // Relative time rendering for use in most screens like Recent conversations.
 //
 // Current date is passed as an argument for unit testing
-export function relative_time_string_from_date({
-    date,
-    current_date = new Date(),
-}: {
-    date: Date;
-    current_date?: Date;
-}): string {
+export function relative_time_string_from_date({date}: {date: Date}): string {
```

Comment:
This could just accept a parameter `date: Date`, rather than accepting an object, right?

Reply:
Updated